core/templates/dev/head/services/AssetsBackendApiServiceSpec.ts

```diff
@@ -65,9 +65,10 @@ describe('Assets Backend API Service', function() {
   it('should correctly formulate the download URL', function() {
     // TODO(sll): Find a way to substitute out constants.DEV_MODE so that we
     // can test the production URL, too.
-    expect(
-      AssetsBackendApiService.getAudioDownloadUrl('expid12345', 'a.mp3')
-    ).toEqual('/assetsdevhandler/expid12345/assets/audio/a.mp3');
+    AssetsBackendApiService.getAudioDownloadUrlAsync('expid12345', 'a.mp3')
```

Comment:
@Jamesjay4199 I just realized something (after I merged) about this test. Change the URL on line 70 to something random and run the frontend tests. Do they pass?

If so, please rework this test to call `done()` after the async call. See https://jasmine.github.io/2.0/introduction.html#section-Asynchronous_Support. Do this for other async calls.

Please address this in a separate follow-up PR.

Reply:
Addressed in https://github.com/oppia/oppia/pull/7244/files